lte/gateway/c/oai/tasks/mme_app/mme_app_bearer.c

```diff
@@ -84,6 +84,119 @@
 #define TASK_SPGW TASK_S11
 #endif
 
+int send_modify_bearer_req(mme_ue_s1ap_id_t ue_id, ebi_t ebi)
+{
+  OAILOG_FUNC_IN(LOG_MME_APP);
+  uint8_t item = 0;
+  struct ue_mm_context_s* ue_context_p = NULL;
+  mme_app_desc_t* mme_app_desc_p = NULL;
+  mme_app_desc_p = get_mme_nas_state(false);
+  ue_context_p = mme_ue_context_exists_mme_ue_s1ap_id(
+    &mme_app_desc_p->mme_ue_contexts, ue_id);
+
+  if (ue_context_p == NULL) {
+    OAILOG_ERROR(
+      LOG_MME_APP,
+      "ue_context_p is NULL, did not send S11_MODIFY_BEARER_REQUEST\n");
+    OAILOG_FUNC_RETURN(LOG_MME_APP, RETURNerror);
+  }
+  MessageDef* message_p =
+    itti_alloc_new_message(TASK_MME_APP, S11_MODIFY_BEARER_REQUEST);
```

Comment:
When we return due to errors before actually sending the itti message, we should free the allocated memory for the itti message.